src/components/toast/toast.scss

```diff
@@ -0,0 +1,128 @@
+@import "mixins";
+
+@mixin inherit-color() {
+  color: inherit;
+  background-color: inherit;
+}
+
+.show {
+  opacity: 1 !important;
```

Comment:
Why is this `!important` required?

Reply:
it was added due to shadowdom, but i will fix it by adding this class to the host selector.